### DISCUSSION TITLE
Broken links on wiki that used to lead to forum, can we point to new forum?

### DISCUSSION THREAD

**Discussion by HippasusTwo** [Tuxemon/Tuxemon#2265] [Q&A]
In particular, I'm thinking about the one talking about maps that need to be done, which I recall being just a broken link. Can someone make a new page full of info on what mapping stuff needs to be done, and other to-dos?

**Comment by Sanglorian** [comment]
Hi @HippasusTwo , good idea. I think that sprint probably listed maps that were missing from the Spyder campaign, and we have all the maps we need for that now. 

I can take a look at any other broken links too, if you point me to them.

**Comment by HippasusTwo** [comment]
@Sanglorian  Alright, so, main page the link to forum (under About The Game) should be changed to point to https://github.com/Tuxemon/Tuxemon/discussions.
On the Game Bible, the 'irc' link under forums is broken. The 'content that is yet to be incorporated' section is ALL forum links except for the last two. Tiles, logos, and overland sprites. Licensing section has another forum link.
The tutorial 'SPRINT 3! getting maps, encounters, and npcs done' is a broken link.
Features 'rumble support' points to a broken page. 

The map section like I said...

Basically, the Game Bible page seems to have tons of broken links, mostly to now lost 'todo' pages. I think I pointed out all of 'em, but I wouldn't be surprised if there was one more. If you could swap most of them with a single link pointing at a 'todo' new forum topic or something, that'd be great. Because right now, anyone looking to help out who visits the Game Bible for an overview is going to be very confused!

**Reply by Sanglorian** [reply]
Thanks! I think I've cleared them all out.
